prbt_hardware_support/test/unittests/unittest_modbus_adapter_brake_test.cpp

```diff
@@ -48,9 +50,10 @@ static const ModbusApiSpec test_api_spec{ {modbus_api_spec::VERSION, 969},
 class ModbusAdapterBrakeTestTest : public testing::Test, public testing::AsyncTest
 {
 public:
-  ModbusAdapterBrakeTestTest();
-  ~ModbusAdapterBrakeTestTest();
+  void SetUp();
+  void TearDown();
```

Comment:
I see no necessary reason to use SetUp()/TearDown() here.
fyi:

> Should I use the constructor/destructor of the test fixture or SetUp()/TearDown()?

in https://github.com/google/googletest/blob/master/googletest/docs/faq.md

Reply:
Did you try to stop the spinner in the destructor? The class members are destroyed after the destructor is called, so this should not be a problem.

Reply:
Done.